Tests/MapboxMapsTests/Style/LightTests.swift

```diff
@@ -1,5 +1,5 @@
 import XCTest
-@_spi(Experimental) @testable import MapboxMaps
```

Comment:
I'm not so sure about this anymore, reading style spec saying that Light has some experimental properties but I couldn't find it in our source code, maybe we haven't synced this. @evil159 @OdNairy do you know anything about this?